pkg/gadgets/trace/signal/types/types.go

```diff
@@ -15,18 +15,23 @@
 package types
 
 import (
+	"github.com/kinvolk/inspektor-gadget/pkg/columns"
 	eventtypes "github.com/kinvolk/inspektor-gadget/pkg/types"
 )
 
 type Event struct {
 	eventtypes.Event
 
-	Pid       uint32 `json:"pid,omitempty"`
-	TargetPid uint32 `json:"tpid,omitempty"`
-	Signal    string `json:"signal,omitempty"`
-	Retval    int    `json:"ret,omitempty"`
-	Comm      string `json:"comm,omitempty"`
-	MountNsID uint64 `json:"mountnsid,omitempty"`
+	Pid       uint32 `json:"pid,omitempty" column:"pid,width:7,fixed"`
+	Comm      string `json:"comm,omitempty" column:"comm,width:16,fixed"`
+	Signal    string `json:"signal,omitempty" column:"signal,width:6,fixed,ellipsis:start"`
+	TargetPid uint32 `json:"tpid,omitempty" column:"tpid,width:7,fixed"`
+	Retval    int    `json:"ret,omitempty" column:"ret,width:3,fixed"`
```

Comment:
For SIGNAL, the most common signals SIGKILL, SIGTERM, SIGINT are 6, so I used that value fixed with `ellipsis:start` as all of them start with `SIG`. However, I think it would be better to have mix=6 and max=11 (SIGRTMIN+13).

```bash
$ tput cols
95
$ ./kubectl-gadget trace signal -A
NODE        NAMESPACE   POD         CONTAINER  PID     COMM             SIGNAL      TPID    RET
master      gadget      gadge…9zn9s gadget     93865   gadgettracerman  SIGURG      106393  0
master      gadget      gadge…9zn9s gadget     93865   gadgettracerman  SIGURG      106393  0
```